src/resuming-subscription.ts

```diff
@@ -108,21 +108,25 @@ export let createResumingStrategy: (retryingOptions: RetryStrategyOptions, initi
     
                         let errorResolution = resolveError(error);
                         if(errorResolution instanceof Retry){
-                            this.timeout = window.setTimeout(() => { executeNextSubscribeStrategy(lastEventId) }, errorResolution.waitTimeMillis);
+                            let subID = errorResolution.subID;
+
+                            this.timeout = window.setTimeout(() => {
+                                executeNextSubscribeStrategy(lastEventId, subID)
```

Comment:
Why do you need subscription ids here? They should be an implementation detail for transports.

Reply:
Because we need to identify what subscription we want to retry (web sockets are multiplexed). I know it's not ideal but without passing subID we can not retry subscriptions properly. (since subscription it's just one message type send through same WS connection)